codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/Rust.kt

```diff
@@ -42,6 +42,43 @@ import java.nio.file.Files.createTempDirectory
 import java.nio.file.Path
 import kotlin.io.path.absolutePathString
 
+// cargo commands and env values
+object Commands {
+    private const val cfgUnstable = "--cfg aws_sdk_unstable"
+    fun func(s: String, add: String, flag: Boolean): String {
+        if (flag) {
+            return s + " " + add
+        } else {
+            return s
+        }
+    }
```

Comment:
This can be simpler in Kotlin:
```suggestion
    fun func(s: String, add: String, flag: Boolean): String = if (flag) { s + " " + add } else { s }
```
Also consider renaming the function to something like `combineFlags`.